mkdocs.yml

```diff
@@ -67,6 +67,34 @@ nav:
   - examples.md
   - faq.md
   - contributing.md
+  - Future documentation (WIP):
+      - Introduction: working/introduction/introduction.md
+      - Installation:
+          - working/installation/install.md
+          - working/installation/upgrade.md
+      - Getting Started: working/getting_started/getting_started.md
+      - User Guide:
+          - working/user_guide/understanding.md
+          - working/user_guide/creating_testing_collection.md
+          - working/user_guide/testing_existing_collection.md
+          - working/user_guide/testing_existing_roles.md
+          - working/user_guid/ci_integration.md
+          - working/user_guid/using_tox-ansible.md
+      - References:
+          - working/references/cli.md
+          - working/references/configuration.md
+          - working/references/porting_change_log.md
+          - working/references/road_map.md
+      - Examples:
+          - working/examples/example_1.md
+          - working/examples/example_2.md
+          - working/examples/example_3.md
+      - Frequently asked questions: working/faq/faq.md
```

Comment:
Consider not having a FAQ but including everything that is in the current FAQ into sections in the user guide.  FAQs tend to expand over time as an easy 'dumping ground' of new information. That said, y'all know your readers better so maybe some of the FAQs really are.. .frequently... repeatedly...asked questions :-)

Reply:
We tend to use the faq as a place to put things that have come in as an issue or question that would only otherwise end up as "Note" or "warning" in the user guide. Users of downstream docs would necessarily have access to the upstream GH issues list so having the FAQ downstream can help.

That said, the point is good. Before adding a new ? to the faq, stop and ask would it be better in the user guide.